### PR TITLE
feat: Add release drafter

### PR DESCRIPTION
**Pull request by Aniket-508**
### Description

- Added Release Drafter by configuring a YAML-based workflow file at .github/workflows/release-drafter.yml which drafts the next release notes as pull requests are merged into the main branch.
- Added Update Changelog by configuring a YAML-based workflow file at .github/workflows/update-changelog.yml which also drafts the new versions keeping previous version changes in changelog.md file.
- Also added the required labels that were provided by repo maintainers under appropriate titles.

### Checklist:
- [x] Followed the guidelines mentioned in our [Contributing document](../CONTRIBUTING.md)
- [x] Ensured that there aren't other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change
- [x] This PR is linked to an [Issue](https://www.github.com/Auto-DL/Generator/issues) and the issue number has been mentioned in the title and body of the PR
- [x] The code is clean and docstrings have been added or updated as required
- [x] Code is linted/formatted locally prior to submission (using `black`)

### CODE REVIEW

File: .github/release_drafter.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'patch'
+  default: patch
+template: |
+  ## What's Changed
+  $CHANGES
```
Comment (RusherRG) [inline .github/release_drafter.yml:42]: @aniket-508 Add a new line at EOF

Comment (RusherRG) [reply .github/release_drafter.yml:42]: Make this one minor change and then we can merge and test

**Review by RusherRG — CHANGES_REQUESTED** [review]
Make that one

**Comment by Aniket-508** [comment]
@RusherRG Done with the change

**Review by RusherRG — APPROVED** [review]

**Pull request merged**